synapse-web-monorepo.code-workspace

```diff
@@ -0,0 +1,31 @@
+{
+  // Shared VS Code settings for synapse-web-monorepo
+  // This file is used to configure the workspace settings and suggested extensions used by all devs
+  // Your preferred overrides can go in .vscode/settings.json
+
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "settings": {
+    // General settings
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+
+    // Jest extension settings
+    "jest.runMode": "on-demand",
+    "jest.rootPath": "packages/synapse-react-client", // synapse-react-client is the only Jest project
+
+    // Vitest extension settings
+    "vitest.maximumConfigs": 99
```

Comment:
Load all vitest projects. All of our vitest suites are small, so I am not super worried about this causing performance issues. If it does, we can decrease it